supervisor/api/backups.py

```diff
@@ -50,24 +55,29 @@
 _ALL_FOLDERS = ALL_FOLDERS + [FOLDER_HOMEASSISTANT]
 
 # pylint: disable=no-value-for-parameter
-SCHEMA_RESTORE_PARTIAL = vol.Schema(
+SCHEMA_RESTORE_FULL = vol.Schema(
     {
         vol.Optional(ATTR_PASSWORD): vol.Maybe(str),
+        vol.Optional(ATTR_BACKGROUND): vol.Maybe(vol.Boolean()),
```

Comment:
Still unsure about not making a new job based API (even if not WebSocket, just another path). Anyway if we doing this, a propose to set the option default:
```suggestion
        vol.Optional(ATTR_BACKGROUND, default=False): vol.Boolean(),
```
Otherwise we would allow False/True and None as value for a control function that have a value and that is False is not set